README.md

```diff
@@ -16,4 +16,4 @@ You are given movies loaded from the API and initial markup. Your task is to:
 - Install Prettier Extention and use this [VSCode settings](https://mate-academy.github.io/fe-program/tools/vscode/settings.json) to enable format on save.
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_movies-list-js/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://IvanRyabukha.github.io/react_movies-list-js/) and add it to the PR description.
```

Comment:
Add this demo link to the PR description 
<img width="971" alt="image" src="https://github.com/user-attachments/assets/59088719-50d5-4e36-8ac6-03f1106d514e" />